modules/boulder/src/main/ts/ephox/boulder/core/Utils.ts

```diff
@@ -0,0 +1,26 @@
+import { SimpleResult } from '../alien/SimpleResult';
+import * as SchemaError from './SchemaError';
+import { StructureProcessor, ValueValidator } from './StructureProcessor';
+
+const value = (validator: ValueValidator): StructureProcessor => {
+  const extract = (path, val) => {
+    return SimpleResult.bindError(
+      validator(val),
+      (err) => SchemaError.custom(path, err)
+    );
+  };
+
+  const toString = () => 'val';
+
+  return {
+    getProp: extract,
+    toString
+  };
+};
+
+const _anyValue: StructureProcessor = value(SimpleResult.svalue);
```

Comment:
Can we leave a comment explaining why this is prefixed with an underscore

Reply:
No idea unfortunately - I just copied it. Java legacy things I guess?

Reply:
Changed the name per suggestion in the call